components/Carousel/styles.js

```diff
@@ -0,0 +1,38 @@
+import { Dimensions, StyleSheet, Platform } from 'react-native';
+
+const { width: screenWidth } = Dimensions.get('window');
```

Comment:
This separate styles sheet helps with the carousel and compartmentalizing the styles. I like how this is prganized.